README.md

```diff
@@ -91,7 +91,7 @@ selectors = {
 }
 
 # Benchmark
-score_df, selected_df, runtime_df = benchmark(selectors, data, label)
+score_df, selected_df, runtime_df = benchmark(selectors, data, label, cv=5)
```

Comment:
I think k = 5 for our CV parameters is a safe bet as a default/example compared to k=10

Reply:
Could this trap DS to run CV = 5 on huge datasets, if they immediately copy/run the usage example? 

That is, should we maybe just leave this out from the example?

Reply:
Agreed. Making cross-validation visible, as is in the example now, is important for the case of Feature Selection.